client/packages/system/src/Patient/Encounter/CreateEncounterModal.tsx

```diff
@@ -107,14 +102,17 @@ export const CreateEncounterModal: FC = () => {
     setDraft({ createdDatetime, ...draft, clinician });
   };
 
+  const canSubmit = () =>
+    draft !== undefined && draft.clinician && draft.startDatetime;
```

Comment:
```suggestion
    draft !== undefined && draft.startDatetime;
```

Think we shouldn't make this required because it might be unknown, e.g. when scheduling an encounter. And the demo server doesn't has clinicians set

Reply:
Sure, no longer required.